E2E-ORS-Sample-For-Java/src/main/java/objectReplicationMonitor.java

```diff
@@ -0,0 +1,329 @@
+import com.azure.storage.blob.*;
+import com.azure.storage.blob.batch.BlobBatchClient;
+import com.azure.storage.blob.batch.BlobBatchClientBuilder;
+import com.azure.storage.blob.batch.BlobBatchStorageException;
+import com.azure.storage.blob.models.*;
+
+import java.util.*;
+import java.io.*;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+
+public class objectReplicationMonitor {
+    /*
+    this class is to demonstrate what object replication does. please read the README in its entirety
+    before running this program and make sure all prerequisites are met.
+     */
+
+    public static void main(String[] args) throws IOException {
+
+        // get path to config file
+        Path currentPath = Paths.get(System.getProperty("user.dir"));
+        Path pathToDir = Paths.get(currentPath.toString(),
+                "src", "main", "java");
+        String pathToConfig = Paths.get(pathToDir.toString(), "app.config").toString();
+
+        // get values from config files
+        InputStream input = new FileInputStream(pathToConfig);
+        Properties prop = new Properties();
+        prop.load(input);
+
+        String sourceContainerName = prop.getProperty("sourceContainerName");
+        String sourceStorageAccountConnectionString = prop.getProperty("sourceStorageAccountConnectionString");
+        String destinationContainerName = prop.getProperty("destinationContainerName");
+        String destinationStorageAccountConnectionString = prop.getProperty("destinationStorageAccountConnectionString");
+        String blobsToReplicate = prop.getProperty("blobsToReplicate");
+        String archiveMethod = prop.getProperty("archiveMethod");
+
+        // creating a sample list of blobs to upload and replicate
+        String[] replicatedBlobList = new String[1000];
+        int num = 0;
+        for (int i = 0; i < replicatedBlobList.length; i++) {
+
```

Comment:
nit: new lines (this applies everywhere else too like by your if statements). Keep new lines restricted to places where you want to delineate different steps.